src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue29558.cs

```diff
@@ -0,0 +1,36 @@
+﻿#if ANDROID
```

Comment:
Add a comment here to allow quickly determinate why only runs on Android in the future.